### PR TITLE
Traditional module multiple select name fix

### PR DESCRIPTION
**Pull request by msx2**
@arendjr I noticed I made a small mistake yesterday, so I'm correcting it now :) `<select multiple>` should have `name` ending with `[]` in order to work properly with traditional forms.

### CODE REVIEW

File: src/selectivity-templates.js [hunk 1]
```diff
@@ -278,6 +278,9 @@ Selectivity.Templates = {
      *                mode - Mode in which select exists, single or multiple.
      */
     selectCompliance: function(options) {
+        if (options.mode === 'multiple' && !options.name.match(/\[\]$/)) {
```
Comment (arendjr) [inline src/selectivity-templates.js]: Why not use `options.name.slice(-1) !== '[]'`? That seems a bit easier for readability (and performance) than trying to parse a regular expression :)

Also, you can shorten the next line to `options.name += '[]'`.

Comment (msx2) [inline src/selectivity-templates.js]: Haha, you are right. Syntax updated, synced with current master version and built. :)

**Comment by arendjr** [comment]
Oops, looks like I misled you with my comment... `.slice(-1)` should be `.slice(-2)` of course. Sorry! :)

**Comment by arendjr** [comment]
Great, thanks!

**Pull request merged**